### ISSUE TITLE
Not working while in background

### ISSUE THREAD

**Issue by cyborg-kaka** [shaileshrkumar/google_meet_attendance_tracker_new-version#1]
If someone left while recorder(who is using this extension) is on other tabs/windows it does not detects that & mark him/her left when the recorder is back !

**Comment by shaileshrkumar** [comment]
> If someone left while recorder(who is using this extension) is on other tabs/windows it does not detects that & mark him/her left when the recorder is back !

True...just now understanding the issue....do you know how to fix it using javascript/jquery
